test/v4/utils.js

```diff
@@ -226,25 +228,25 @@ contract('utils', function(accounts) {
 function calcDestQty (srcQty, rate, srcDecimal, dstDecimal) {
     let result;
     if (dstDecimal >= srcDecimal) {
-        result = ((((new BigNumber(srcQty)).mul(rate).mul((new BigNumber(10)).pow(dstDecimal - srcDecimal))).div(PRECISION)));
+        result = (((new BN(srcQty)).mul(rate).mul((new BN(10)).pow(new BN(dstDecimal - srcDecimal)))).div(PRECISION));
     } else {
-        result = ((new BigNumber(srcQty)).mul(rate).div(PRECISION.mul((new BigNumber(10)).pow(srcDecimal - dstDecimal))));
+        result = (new BN(srcQty)).mul(rate).div(PRECISION.mul((new BN(10)).pow(new BN(srcDecimal - dstDecimal))));
     }
-    return result.floor();
+    return Math.floor(result)
 }
 
 function calcSrcQty(dstQty, srcDecimals, dstDecimals, rate) {
     //source quantity is rounded up. to avoid dest quantity being too low.
     let numerator;
     let denominator;
     if (srcDecimals >= dstDecimals) {
-        numerator = PRECISION.mul(dstQty).mul((new BigNumber(10)).pow(srcDecimals - dstDecimals));
```

Comment:
please move to helper file.

Reply:
Moved to helper file, please use that thanks =)